blocks/library/paragraph/index.js

```diff
@@ -311,6 +311,10 @@ const schema = {
 	customFontSize: {
 		type: 'number',
 	},
+	canInsertTokens: {
```

Comment:
It probably shouldn't be a part of `attributes`, because their role is to help with parsing block's definition into data to consume. It should be rather inside `supports` (see https://github.com/WordPress/gutenberg/blob/master/docs/block-api.md#supports-optional) property or directly inside `settings` part of block's definition.

Reply:
Makes sense, thanks. I don't currently see a way to access the `supports` or `settings` in the editor store. Perhaps this is a sign that I need to add new Redux actions? The design discussion happening now as to where the button to add inline images should be may have some implications here as well.

Reply:
The store only tracks the name, UUID, and current attribute state of a block. But the name is enough to get the supports of a block, via [`getBlockSupport( name, feature, defaultSupports )`](https://github.com/WordPress/gutenberg/blob/2ae21083a8deb71685523a6de35647b774ba351f/blocks/api/registration.js#L258-L276), [`hasBlockSupport( name, feature, defaultSupports )`](https://github.com/WordPress/gutenberg/blob/2ae21083a8deb71685523a6de35647b774ba351f/blocks/api/registration.js#L278-L290), or simply [`getBlockType( name ).supports`](https://github.com/WordPress/gutenberg/blob/2ae21083a8deb71685523a6de35647b774ba351f/blocks/api/registration.js#L238-L247).

(Aside: Wondering if these ought to be called `getBlockTypeSupport`, `hasBlockTypeSupport` for terminology consistency)

Reply:
Ah, thanks for pointing out those functions! I hadn't noticed them yet and they should be useful.